contrib/dist/linux/buildrpm.sh

```diff
@@ -267,11 +276,16 @@ fi
 # Find where the top RPM-building directory is
 #
 
-rpmtopdir=
-file=~/.rpmmacros
-if test -r $file; then
-    rpmtopdir=${rpmtopdir:-"`grep %_topdir $file | awk '{ print $2 }'`"}
+# if the user did not specify an $rpmtopdir, check for an .rpmmacros file.
+if test "$rpmtopdir" == ""; then
```

Comment:
Actually, I think the use of `==` is wrong here.  Sorry we missed it on the master PR.  😦 

I think it works with Linux `test`, but it fails (IIRC) with BSD flavors of `test`.

It might be a little better to `if test -z "$rpmtopdir"; then` ...

Reply:
Actually, I think this is moot -- this script is for building RPMs.  Which is specific to Linux -- not BSD.  So the use of `==` is probably ok here.

I therefore retract my prior comment.  😄

Reply:
*phew*